tests/mmm-ext-deposit.spec.ts

```diff
@@ -0,0 +1,605 @@
+        connection,
+        wallet.payer,
+      );
+      const { mint, recipientTokenAccount } =
+        await createTestMintAndTokenT22VanillaExt(
+          connection,
+          wallet.payer,
+          undefined,
+          groupKeyPair,
+        );
+
+      const poolData = await createPool(program, {
+        owner: wallet.publicKey,
+        cosigner,
+        allowlists: [
+          {
+            kind: AllowlistKind.any,
+            value: PublicKey.default,
+          },
+          ...getEmptyAllowLists(5),
+        ],
+      });
+
+      const poolAta = await getAssociatedTokenAddress(
+        mint,
+        poolData.poolKey,
+        true,
+        TOKEN_2022_PROGRAM_ID,
+      );
+
+      const { key: sellState } = getMMMSellStatePDA(
+        program.programId,
+        poolData.poolKey,
+        mint,
+      );
+
+      assert.equal(await connection.getBalance(poolAta), 0);
+      assert.equal(await connection.getBalance(sellState), 0);
+      let poolAccountInfo = await program.account.pool.fetch(poolData.poolKey);
+      assert.equal(poolAccountInfo.sellsideAssetAmount.toNumber(), 0);
+
+      await program.methods
+        .extDepositSell({
+          assetAmount: new anchor.BN(1),
+          allowlistAux: '',
+        })
+        .accountsStrict({
+          owner: wallet.publicKey,
+          cosigner: cosigner.publicKey,
+          pool: poolData.poolKey,
+          assetMint: mint,
+          assetTokenAccount: recipientTokenAccount,
+          sellsideEscrowTokenAccount: poolAta,
+          sellState,
+          systemProgram: SystemProgram.programId,
+          tokenProgram: TOKEN_2022_PROGRAM_ID,
+          associatedTokenProgram: ASSOCIATED_TOKEN_PROGRAM_ID,
+        })
+        .signers([cosigner])
+        .rpc({ skipPreflight: true });
+
+      let nftEscrow = await getTokenAccount2022(
+        connection,
+        poolAta,
+        TOKEN_2022_PROGRAM_ID,
+      );
+      assert.equal(Number(nftEscrow.amount), 1);
+      assert.equal(nftEscrow.owner.toBase58(), poolData.poolKey.toBase58());
+      poolAccountInfo = await program.account.pool.fetch(poolData.poolKey);
+      assert.equal(poolAccountInfo.sellsideAssetAmount.toNumber(), 1);
+      assert.equal(await connection.getBalance(recipientTokenAccount), 0);
+
+      const sellStateAccountInfo = await program.account.sellState.fetch(
+        sellState,
+      );
+      assert.equal(
+        sellStateAccountInfo.pool.toBase58(),
+        poolData.poolKey.toBase58(),
+      );
+      assert.equal(
+        sellStateAccountInfo.poolOwner.toBase58(),
+        wallet.publicKey.toBase58(),
+      );
+      assert.equal(sellStateAccountInfo.assetMint.toBase58(), mint.toBase58());
+      assert.equal(sellStateAccountInfo.assetAmount.toNumber(), 1);
+      assert.deepEqual(
+        sellStateAccountInfo.cosignerAnnotation,
+        new Array(32).fill(0),
+      );
+
+      const { mint: mint2, recipientTokenAccount: recipientTokenAccount2 } =
```

Comment:
can probably remove this second test, or test with no group

Reply:
oh no_group has been tested in `correctly verifies depositing nfts with ANY allowlist`.